tests/security/ansible/test.sh

```diff
@@ -9,10 +9,22 @@ rlJournalStart
         rlRun "pushd data"
     rlPhaseEnd
 
-    rlPhaseStartTest
-        rlRun -s "tmt run --id $run -vvvddd" 1-3
-        rlAssertNotGrep "revealed" $rlRun_LOG
-    rlPhaseEnd
+    for method in ${METHODS:-container}; do
+        rlPhaseStartTest "Test ($method)"
+            rlRun -s "tmt run --id $run -avvvddd provision -h $method" 1-3
+            rlAssertNotGrep "revealed" $rlRun_LOG
+
+            # For container provision try centos images as well
```

Comment:
This section is probably not needed. In the `/tests/prepare/ansible` it is used to confirm that `ansible` is correctly working with older distros. Here we just want to make sure the `ansible` command line is sane and is correctly processed.